source/includes/steps-tutorial-react-native-2.yaml

```
@@ -221,15 +116,16 @@ content: |
   descendants can use. As the name implies, the TasksProvider provides the task
   objects for a given project. The project partition is passed in as a prop.
 
-  The first thing to implement is the opening of the realm for the given
-  project. Whereas the user realm used the partition key value pattern of
-  ``user=<USER_ID>``, projects use the partition key value pattern of
+  The first thing to implement is opening the realm for the specified
+  project. When configuring the user realm, we used a partition key value pattern of
+  ``user=<USER_ID>``; similarly, projects use a partition key value pattern of
   ``project=<USER_ID>``. For the sake of this tutorial's simplicity, each user
   always has exactly one project, specified by the partition key value
   containing their own user ID.
 
-  In the effect block, open the realm. Once opened, attach an observer that will
-  update the tasks list when any changes come in:
+  In ``providers/TasksProvider.js``, in the effect block, configure and open the 
+  realm. Once opened, attach an observer that will update the tasks list when 
```

suggestion: simplify

```suggestion
  In the ``effect`` block of ``providers/TasksProvider.js``, configure and open the 
  realm. Once opened, attach an observer that will update the tasks list when 
```